src/internal/views/nunjucks/internal-search/macros/company-licences.njk

```diff
@@ -20,6 +21,13 @@
               <a class="govuk-link" href="/licences/{{licence.documentId}}">{{licence.licenceRef}}</a>
             </td>
             <td class="govuk-table__cell">{{licence.licenceHolder}}</td>
+
+            <td class="govuk-table__cell govuk-table__cell--numeric">
+              {% if userCanUnlinkLicences %}
+                <a class="govuk-link" href="/licences/{{licence.documentId}}/{{unlinkLicencePathTail}}">Unlink licence</a>
```

Comment:
For accessibility, we may want something like:
```
Unlink licence<span class="govuk-visually-hidden"> {{ licence.licenceNumber }}</span>
```
This avoids having multiple links on the page with the same text but pointing to a different page